### ISSUE TITLE
SmartCity-Accessibility Server Side Unit Tests #723 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#385]
Build 'SmartCity-Accessibility Server Side Unit Tests' is failing!

Last 50 lines of build :
```
[...truncated 22.58 KB...]
	at com.teamdev.jxmaps.internal.internal.SharedMemoryLibrary.a(SourceFile:28)
	at com.teamdev.jxmaps.internal.internal.ipc.e.a(SourceFile:56)
	at com.teamdev.jxmaps.internal.internal.ipc.p.run(SourceFile:183)
	at java.lang.Thread.run(Thread.java:748)
Tests run: 3, Failures: 0, Errors: 2, Skipped: 1, Time elapsed: 1,000.021 sec <<< FAILURE! - in smartcity.accessibility.search.SearchTest
test2(smartcity.accessibility.search.SearchTest)  Time elapsed: 499.994 sec  <<< ERROR!
com.teamdev.jxmaps.internal.internal.ipc.v: Failed to initialize IPC. Remote process doesn't respond.
	at smartcity.accessibility.search.SearchTest.test2(SearchTest.java:57)

test3(smartcity.accessibility.search.SearchTest)  Time elapsed: 500.013 sec  <<< ERROR!
com.teamdev.jxmaps.internal.internal.ipc.v: Failed to initialize IPC. Remote process doesn't respond.
	at smartcity.accessibility.search.SearchTest.test3(SearchTest.java:75)


Results :

Tests in error: 
  NearbyPlacesTest.nearByPlacesTest:32 » v Failed to initialize IPC. Remote proc...
  SearchTest.test2:57 » v Failed to initialize IPC. Remote process doesn't respo...
  SearchTest.test3:75 » v Failed to initialize IPC. Remote process doesn't respo...
  AdressSearchServiceTest.searchSuccess » IllegalState Failed to load Applicatio...
  LocationsInRadiusServiceTest.locationtest1 » IllegalState Failed to load Appli...
  LoginServiceTest.UserInfoSuccess » IllegalState Failed to load ApplicationCont...
  LoginServiceTest.loginSuccess » IllegalState Failed to load ApplicationContext
  LoginServiceTest.loginUserDoesNotExist » IllegalState Failed to load Applicati...
  LoginServiceTest.signupSuccess » IllegalState Failed to load ApplicationContex...
  LoginServiceTest.signupUserExist » IllegalState Failed to load ApplicationCont...
  UserInformationServiceTest.searchSuccess » IllegalState Failed to load Applica...

Tests run: 72, Failures: 0, Errors: 11, Skipped: 2

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 26:12 min
[INFO] Finished at: 2017-05-27T14:37:16+00:00
[INFO] Final Memory: 31M/202M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [KoralChapnik] 90e1c09a6b792c5a791dfff96d16d3f1946c9e69 - some changes together #382

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Unit%20Tests/723/)

**Comment by koralchapnik** [comment]
@KaplanAlexander I don't really understand it.. when I run the tests all of them pass

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility Server Side Unit Tests' is failing!

Last 50 lines of build :
```
[...truncated 21.39 KB...]
lat is : 32.067312 lng is : 34.7781
lat is : 32.061621 lng is : 34.764118
lat is : 31.901406 lng is : 35.200473
lat is : 31.777545 lng is : 35.221896
lat is : 32.060276 lng is : 34.764359
lat is : 32.078398 lng is : 34.78706
lat is : 32.061957 lng is : 34.783662
lat is : 31.784352 lng is : 35.117952
lat is : 31.778306 lng is : 35.216163
lat is : 32.124488 lng is : 34.825184
lat is : 32.108255 lng is : 34.838057
lat is : 32.069621 lng is : 34.793664
lat is : 32.077567 lng is : 34.789139
lat is : 32.114059 lng is : 34.824801
arrived to search on complete
Tests run: 3, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.239 sec - in smartcity.accessibility.search.SearchTest

Results :

Tests in error: 
  AdressSearchServiceTest.searchSuccess » IllegalState Failed to load Applicatio...
  LocationsInRadiusServiceTest.locationtest1 » IllegalState Failed to load Appli...
  LoginServiceTest.UserInfoSuccess » IllegalState Failed to load ApplicationCont...
  LoginServiceTest.loginSuccess » IllegalState Failed to load ApplicationContext
  LoginServiceTest.loginUserDoesNotExist » IllegalState Failed to load Applicati...
  LoginServiceTest.signupSuccess » IllegalState Failed to load ApplicationContex...
  LoginServiceTest.signupUserExist » IllegalState Failed to load ApplicationCont...
  UserInformationServiceTest.searchSuccess » IllegalState Failed to load Applica...

Tests run: 72, Failures: 0, Errors: 8, Skipped: 2

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:13 min
[INFO] Finished at: 2017-05-27T14:56:06+00:00
[INFO] Final Memory: 35M/211M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [KoralChapnik] 90e1c09a6b792c5a791dfff96d16d3f1946c9e69 - some changes together #382

- [KoralChapnik] 1639183542d49282818c6463532a930a0c39242d - only needs to add details for locations in client side #382



[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Unit%20Tests/725/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
